src/antsibull_changelog/cli.py

```diff
@@ -609,7 +616,8 @@ def command_release(args: Any) -> int:
     version, codename = _get_version_and_codename(
         paths, config, collection_details, args
     )
-
+    if args.update_galaxy_file and paths.galaxy_path:
```

Comment:
If `galaxy_path` is None but `--update-galaxy-file` is passed, this should raise an error instead of continuing silently.

Reply:
I think it should `return 1` and exit with an error code, not just print a log message.